.github/workflows/main.yml

```diff
@@ -54,26 +62,26 @@ jobs:
         libzstd-dev
 
         pip install
-        meson==0.55.3
+        meson==0.56.2
```

Comment:
Maybe the pinned meson version should also go to `env`?

Reply:
Done!